### ISSUE TITLE
Action does not work anymore (set-env is deprecated)

### ISSUE THREAD

**Issue by mlshv** [nelonoel/branch-name#6]
Please see https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/

<img width="1451" alt="Снимок экрана 2020-11-17 в 13 10 53" src="https://user-images.githubusercontent.com/16861579/99376709-563eca00-28d6-11eb-9f16-2ef0c18c07be.png">

**Comment by kstolte** [comment]
### TL;DR;
set your version from `@v1` to `@v1.0.1`.

### Long Story short, short story long.
This is now working. 
Usually when you have `nelonoel/branch-name@v1` you would get subsequent patches and minor updates of that are of major version 1.x.x.
I believe that there since there is a tag called `v1` it is not following the usual GitHub actions semvar stuff.

by setting your YML file to pin to the exact version of v1.0.1 it will force the latest release to be used and the action will not throw the error as seen above.

snippet of my known working yml config.
```yml
    - name: Get Branch Name
      uses: nelonoel/branch-name@v1.0.1
```

Down sides to this right now is if there are new versions released we will have to manually move the pin accordingly.

I am not sure if deleting the existing tag of `v1` and creating a new tag of `v1.0.0` would allow the GitHub action semver stuff to work appropriately.

**Comment by bosskmk** [comment]
```
// This doesn't work.
$BRANCH_NAME

// Is this right?
${{ env.BRANCH_NAME }}
```

**Comment by kstolte** [comment]
hi @bosskmk I think that is a separate issue. 

it depends on where you are using the environment variable. 

https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables is the link for the GitHub Actions usage.


I believe if you are using it in a shell text you can do something like `"${env.BRANCH_NAME}"`

**Comment by nelonoel** [comment]
Thanks @mlshv for reporting. Yeah, this has been fixed in `v1.0.1` as @kstolte mentioned. I just updated the README to explicitly use the correct version.
